health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/HeatMap/Map.js

```
@@ -0,0 +1,369 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { scaleQuantile } from "d3-scale";
+import { ZoomableGroup, Geographies, ComposableMap, Geography } from "react-simple-maps";
+import { getTitleHeading } from "../../../utils/locale";
+import { Button } from "@egovernments/digit-ui-components";
+import { getBoundaryTypeByLevel } from "../../../utils/getBoundaryTypeByLevel";
+const Map = ({
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)